packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -388,6 +389,28 @@ export class TelemetryContext implements ITelemetryContext {
 		this.telemetry.set(`${prefix}${property}`, value);
 	}
 
+	/**
+	 * {@inheritDoc @fluidframework/runtime-definitions#ITelemetryContext.append}
+	 */
+	push(prefix: string, property: string, value: TelemetryEventPropertyType[]): void {
+		const prevValue = this.telemetry.get(`${prefix}${property}`);
+		if (typeof prevValue === "string") {
+			try {
+				const values = JSON.parse(prevValue) as TelemetryEventPropertyType[];
+				values.push(...value);
+				this.telemetry.set(`${prefix}${property}`, JSON.stringify(values));
+				return;
+			} catch (err) {}
```

Comment:
I think we want to overwrite the value, or maybe convert to [<old value>, <new value>] state, not <old value" state (catch flow, i.e. we deal not with array)